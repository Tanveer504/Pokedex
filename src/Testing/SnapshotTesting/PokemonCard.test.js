import renderer from 'react-test-renderer';
import PokemonCard from '../../components/Pokemon/PokemonCard'

it('should have the ability component snapshot', () => {
    const tree = renderer.create(<PokemonCard />).toJSON();
    expect(tree).toMatchSnapshot();
});