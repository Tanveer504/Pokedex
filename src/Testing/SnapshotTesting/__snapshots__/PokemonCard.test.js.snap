// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have the ability component snapshot 1`] = `
<div
  className="card-container undefined"
>
  <div
    className="card-container_image"
  >
    <a
      href="/undefined"
      onClick={[Function]}
    >
      <img />
    </a>
  </div>
  <div
    className="name"
  >
    <h3 />
    <h6>
      00
    </h6>
  </div>
</div>
`;
